app/models/article.rb

```diff
@@ -629,6 +634,16 @@ def canonical_url_must_not_have_spaces
     errors.add(:canonical_url, "must not have spaces")
   end
 
+  def user_mentions_in_markdown
+    return if created_at.present? && created_at.before?(MAX_USER_MENTION_LIVE_AT)
+
+    # The "comment-mentioned-user" css is added by Html::Parser#user_link_if_exists
+    mentions_count = Nokogiri::HTML(processed_html).css(".comment-mentioned-user").size
```

Comment:
At some point, it might be nice to refactor the naming here to use the class `.mentioned-user` since it's for comments and articles now.

Reply:
Hah, look at what I have in my editor right now (as a `FIXME` to come back to) 😂 

<img width="774" alt="Screen Shot 2021-04-05 at 4 32 03 PM" src="https://user-images.githubusercontent.com/6921610/113638798-893aed80-962c-11eb-9dee-e7128162d49d.png">

I'm not going to mess with it in this PR, but you can expect it to change in an upcoming PR (am breaking this otherwise chunky RFC up into more management pieces)